Sitemap/Url/GoogleVideoUrlDecorator.php

```diff
@@ -257,7 +258,7 @@ public function setThumbnailLoc($thumbnail_loc)
     {
         $this->bc(__FUNCTION__);
 
-        $this->video->setThumbnailLoc($thumbnail_loc);
+        $this->video->setThumbnailLocation($thumbnail_loc);
```

Comment:
You should rename the parameters as well

Reply:
Do you mean changing the parameters names ? I won't, these methods bow exists only to be removed in the next version. I didn't want to lose some time with this.